src/lobby.cpp

```diff
@@ -563,6 +563,11 @@ void Lobby::set_player_count(int players_online, int max_players)
   ui_player_count->setText(f_string);
 }
 
+void Lobby::set_server_description(const QString& server_description)
+{
+    ui_description->append_linked(server_description);
```

Comment:
```suggestion
    ui_description->clear();
    ui_description->append_linked(server_description);
```
Then go into `on_server_list_clicked` and replace the same set of lines with a call to `set_server_description`